network/src/connectivity_manager/mod.rs

```diff
@@ -343,7 +341,14 @@ where
             .filter(|peer_id| !eligible.contains_key(peer_id))
             .cloned()
             .collect();
+
         for p in stale_dials.into_iter() {
+            debug!(
+                NetworkSchema::new(&self.network_context).remote_peer(&p),
+                "{} Cancelling stale dial {}",
+                self.network_context,
+                p.short_str()
+            );
```

Comment:
I don't have context on what "network_context" display's as, but this is a good instance where it's valuable to provide a fmt message with a short peer_id, and include the full peer id as structured data.

Reply:
network context displays as `[network_id, role, peer_id]`.